src/spurt/graph/_delaunay.py

```diff
@@ -51,18 +59,128 @@ def points(self) -> np.ndarray:
         return self._d.points
 
     @property
-    def simplices(self) -> np.ndarray:
+    def cycles(self) -> np.ndarray:
         return self._d.simplices
 
     @property
     def boundary(self) -> np.ndarray:
         return self._d.convex_hull
 
+    def create_links(self) -> None:
```

Comment:
Shall we make this a private method by prefixing its name with an underscore?

```suggestion
    def _create_links(self) -> None:
```

Reply:
(And likewise for `Reg2DGraph.create_links()`)